docroot/profiles/sdss/sdss_profile/config/sync/system.menu.utility-navigation.yml

```diff
@@ -0,0 +1,8 @@
+uuid: d946876c-ea16-4d3f-a3d5-28af32c2b68b
+langcode: en
+status: true
+dependencies: {  }
+id: utility-navigation
+label: 'Utility Navigation'
+description: 'Suplemental site navigation'
```

Comment:
@sethprime stick another P in supplemental and I will merge it!

Reply:
@buttonwillowsix Thanks! Stuck another P in supplemental.